### PR TITLE
Update Gratwurst.lua to allow slash gw to also open config window.

### LINKED ISSUE #3: Add slash command to open configuration

### LINKED ISSUE #13: Curseforge changelog contains emails
When curseforge packages the add-on they scrape git commit message as the changelog.txt. I don't see the benefit of that existing. I'm not sure how to remove it.

### PR DESCRIPTION
**Pull request by bitobrian**

### CODE REVIEW

File: Gratwurst.lua [hunk 1]
```diff
@@ -196,6 +196,7 @@ local function slashcmd(msg, editbox)
 		DEFAULT_CHAT_FRAME:AddMessage("|cffffedbaGratwurst:|r Slash commands:")
 		DEFAULT_CHAT_FRAME:AddMessage("|cffffedbaGratwurst:|r   /gw enable   -> Enable Gratwurst")
 		DEFAULT_CHAT_FRAME:AddMessage("|cffffedbaGratwurst:|r   /gw disable  -> Disable Gratwurst")
+		InterfaceOptionsFrame_OpenToCategory("Gratwurst")
```
Comment (jimeherr) [inline Gratwurst.lua:201]: I would add a message like above, for /gw config -> Opens Configuration.

Then, and an if check on (message == config) and add this line.

**Review by jimeherr — CHANGES_REQUESTED** [review]

File: Gratwurst.lua (hunk 1 above)
Comment (bitobrian) [reply Gratwurst.lua:201]: we could do 'c'. anything longer loses value imo.

Comment (jimeherr) [reply Gratwurst.lua:201]: we could, but from a user's perspective, 'c' could be a bit vague.   either way is fine by me,

Comment (bitobrian) [reply Gratwurst.lua:201]: Tukui uses that so it's the only good example I could think of. It takes ESC and two clicks to get to it manually. I was just trying to consider a quicker alternative.

**Comment by bitobrian** [comment]
Updated and added a fix for packaging as well.

**Comment by bitobrian** [comment]
Figured it out: Override compilation of repository commit messages with a manually-written change log.

I overlooked it in the curseforge docs.

**Review by jimeherr — APPROVED** [review]

**Pull request merged**